erpnext/agriculture/doctype/plant_batch/plant_batch.py

```diff
@@ -52,6 +54,22 @@ def append_to_child(self, obj_to_append):
 
 		self.save()
 
+	def validate_project(self):
+		if self.project:
+			doc = frappe.get_doc("Project", self.project)
+			doc.expected_start_date = self.start_date
```

Comment:
Do we want to override the project start date in case it's been manually changed? On every save of Plant Batch?

Reply:
we want to change it when it's manually changed.

Reply:
@Alchez  It need to be change once it start date manually change in plant batch